data/strings/strings.txt

```diff
@@ -1835,6 +1797,80 @@
     zh-Hans = 选取地图应该下载的位置
     zh-Hant = 請選擇下載地圖後要放的位置
 
+  [maps_storage_downloaded]
+    comment = E.g. "Downloaded maps: 500Mb" in Maps Storage settings
+    tags = android
+    en = Downloaded maps
+    ar = الخرائط
+    be = Мапы
+    bg = Карти
+    cs = Mapy
+    da = Kort
+    de = Karten
+    el = Χάρτες
+    es = Mapas
+    eu = Mapak
+    fa = نقشه ها
+    fi = Kartat
+    fr = Cartes
+    he = מפות
+    hu = Térképek
+    id = Peta
+    it = Mappe
+    ja = マップ
+    ko = 지도
+    nb = Kart
+    nl = Kaarten
+    pl = Mapy
+    pt = Mapas
+    pt-BR = Mapas
+    ro = Hărţi
+    ru = Загруженные карты
+    sk = Mapy
+    sv = Kartor
+    th = แผนที่
+    tr = Haritalar
+    uk = Завантаженнi мапи
+    vi = Bản đồ
+    zh-Hans = 地图
+    zh-Hant = 地圖
+
+  [maps_storage_internal]
+    comment = Internal storage type in Maps Storage settings (not accessible by the user)
+    tags = android
+    en = Internal hidden storage
```

Comment:
Any suggestions of better alternatives?

I proposed "hidden" because it signals to the user that the path is not accessible.
E.g. "Internal private storage" is more confusing imho, because it implies that the data is accessible to the user still (and only the user).

Reply:
Private also means secure. That's better than "hidden".

Reply:
Changed to "private".
Though I couldn't find a good translation to ru ("частное"/"личное"/"защищённое" плохо подходят, имо - оставил "скрытое").

Reply:
приватное